tests/utils/atomWithObservable.test.tsx

```diff
@@ -37,11 +37,11 @@ it('writable count state', async () => {
     const observable = new Observable<number>((subscriber) => {
       subscriber.next(1)
     })
-    const subject = new Subject<number>()
+    const subject = new ReplaySubject<number>()
     // is this usual to delay the subscription?
```

Comment:
By changing from `Subject` to `ReplaySubject`, this comment is no longer valid or is it still?

Reply:
I'm not sure. If the `setTimeout` is removed, it should still work, but then the value is immediately available on the first subscription instead of after a small delay, which changes the test case a little bit. I would then behave similar to a `BehaviourSubject`.

Reply:
Okay, I'm asking a little differently. This test is probably what I made without knowing much about observable usages. So, the question is if this reflects a typical usage, or at least something we should cover. Adding more tests is also welcome.

In any case, we can work on a new PR.

Reply:
A more typical usage would be the following:
```typescript
const observableAtom = atomWithObservable(() => {
  const subject = new ReplaySubject<number>()
  subject.next(1);
  return subject
})
```

> In any case, we can work on a new PR.

+1. I'm happy to help you with that. I worked a lot with Angular in the past, so I have some experience with RXJS.